test/controllers/authentications_controller_test.rb

```diff
@@ -38,7 +46,7 @@ class AuthenticationsControllerTest < ActionDispatch::IntegrationTest
     assert_equal 'Token is not found', JSON.parse(response.body)['errors'].first['detail']
   end
 
-  test 'when logging out with an access token in a cookie' do
+  test 'when logging out successfully the cookie is destroyed' do
```

Comment:
This test has been refactored to simply check that the cookie containing the access token is destroyed on logout.